sdk/go/common/workspace/creds.go

```diff
@@ -128,6 +131,42 @@ type TokenInformation struct {
 	Name         string `json:"name"`                   // The name of the token.
 	Organization string `json:"organization,omitempty"` // If this was an organization token, the organization it was for.
 	Team         string `json:"team,omitempty"`         // If this was a team token, the team it was for.
+	ExpiresAt    int64  `json:"expiresAt,omitempty"`    // The epoch time (in seconds) when this token expires.
```

Comment:
Nit: improve the field name instead?
```suggestion
	ExpiresAtEpochSeconds    int64  `json:"expiresAt,omitempty"`
```